cargo-shuttle/src/lib.rs

```diff
@@ -832,6 +837,38 @@ impl Shuttle {
         Ok(CommandOutcome::Ok)
     }
 
+    async fn logs_beta(&self, args: LogsArgs) -> Result<CommandOutcome> {
```

Comment:
will there be a non-streaming option?

Reply:
Yes, and probably sooner rather than later since I've just discovered the cost of a Kinesis Stream :smiling_face_with_tear: